java25/hello-img-gradle/{{cookiecutter.project_name}}/HelloWorldFunction/gradle/lambda-build-init.gradle

```diff
@@ -54,7 +54,7 @@ def assertExpectedBuildDir(p) {
 }
 
 def createArtifactDir(project) {
-    def distsDir = project.buildDir.toPath().resolve(project.distsDirName).resolve('lambda-build')
+    def distsDir = project.buildDir.toPath().resolve('distributions').resolve('lambda-build')
```

Comment:
Was this change intentional or auto-generated? If intentional, is it related to the Gradle update?